Client/mods/deathmatch/logic/CClientPlayerVoice.cpp

```diff
@@ -398,6 +398,30 @@ bool CClientPlayerVoice::IsFxEffectEnabled(uint uiFxEffect)
     return m_EnabledEffects[uiFxEffect] ? true : false;
 }
 
+bool CClientPlayerVoice::SetFxEffectParameters(uint uiFxEffect, void* params)
+{
+    if (uiFxEffect >= NUMELMS(m_EnabledEffects))
+        return false;
+
+    if (m_pBassPlaybackStream)
+        if (BASS_FXSetParameters(m_FxEffects[uiFxEffect], params))
+            return true;
+
+    return false;
+}
+
+bool CClientPlayerVoice::GetFxEffectParameters(uint uiFxEffect, void* params)
+{
+    if (uiFxEffect >= NUMELMS(m_EnabledEffects))
+        return false;
+
+    if (m_pBassPlaybackStream)
+        if (BASS_FXGetParameters(m_FxEffects[uiFxEffect], params))
+            return true;
+
+    return false;
+}
```

Comment:
Is there any reason why this code can't be used for player voices yet? If so, what's left to do or know to support player voices?

Reply:
He said somewhere that player voice uses speex or what, not BASS.

Reply:
Then I don't see the purpose of adding code that won't ever work, without obvious indications or comments here that these methods are not finished. You need to read the Lua functions definitions to understand that these methods actually do not work.

I guess there's not an easy way to set BASS sound effect parameters for speex/whatever voice streams. I can imagine that would require rerouting the sound to BASS or making some compatibility layer. Those things sound complex enough to warrant them being implemented in another PR, IMO.